binder/src/main/java/io/grpc/binder/SecurityPolicies.java

```diff
@@ -97,6 +102,42 @@ public static SecurityPolicy hasSignature(
         packageManager, packageName, ImmutableList.of(requiredSignature));
   }
 
+  /**
+   * Creates a {@link SecurityPolicy} which checks if the given package is signed with the same
+   * certificate that the current Android platform was signed with.
+   *
+   * @param packageName the package name of the allowed package.
+   * @param backgroundExecutor an executor suitable for blocking disk I/O.
```

Comment:
Re: "disk" How about just ".. suitable for blocking I/O" since I'm pretty sure PackageManager keeps all its state in memory so we're actually talking about the binder IPC I/O

Reply:
Agreed. Updated.